src/components/controls.vue

```diff
@@ -15,7 +15,6 @@
 
     <b-list-group flush class="overflow-auto w-auto">
       <b-list-group-item v-for="(control, index) in controlItems"
-        v-if="control.label.toLowerCase().includes(filterQuery.toLowerCase())"
```

Comment:
This was flagged by eslint as an error following the updates. The suggestion was to fix the computed property to do the filtering for us. Since this is the only place that property is used the fix below was simple to make.